docs/source/deploy_chaincode.md

```diff
@@ -369,7 +369,7 @@ The chaincode is built by the peer when the chaincode is installed. The install
 
 After you install the chaincode package, you need to approve a chaincode definition for your organization. The definition includes the important parameters of chaincode governance such as the name, version, and the chaincode endorsement policy.
 
-The set of channel members who need to approve a chaincode before it can be deployed is governed by the `Application/Channel/lifeycleEndorsement` policy. By default, this policy requires that a majority of channel members need to approve a chaincode before it can used on a channel. Because we have only two organizations on the channel, and a majority of 2 is 2, we need approve a chaincode definition of asset-transfer (basic) as Org1 and Org2.
+The set of channel members who need to approve a chaincode before it can be deployed is governed by the `Application/Channel/lifecycleEndorsement` policy. By default, this policy requires that a majority of channel members need to approve a chaincode before it can used on a channel. Because we have only two organizations on the channel, and a majority of 2 is 2, we need approve a chaincode definition of asset-transfer (basic) as Org1 and Org2.
```

Comment:
need to change this from: '...before it can used on a channel'
to: '...before it can be used on a channel'

need to insert 'be'

Reply:
Already fixed.